main.go

```diff
@@ -686,3 +711,19 @@ func checkRoot() error {
 
 	return nil
 }
+
+func validateSource(source string) error {
+	if source == "" {
+		return nil
+	}
+
+	r, err := regexp.Compile(`^oci:|dir:|file:`)
```

Comment:
what about docker? https://github.com/kairos-io/kairos-agent/blob/1170576dcce4018a3dffddf500295c3ac3a64a46/pkg/types/v1/common.go#L30

Reply:
and somehow it feels like these two checks should be related instead of having to set at each file

Reply:
I wondered about `docker` too but that's how it was before: https://github.com/kairos-io/kairos-agent/pull/152/files/5b9d043aa0986ea31a3765932d85a5412fda49ac#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261L115

Not sure if `docker` and `oci` are handled differently. I can try to fix this here but it's a change irrelevant to this PR (it looks innocent but you never know :D )

Reply:
yeah good question, for me the main issue is that we have two places which do the validation and that they don't match ... don't want to slow the release of the patch version so definitely ok with keeping it as is

Reply:
docker was the old one, its valid, but not recommended as it doesnt necessary has to be a docker artifact, oci artifacts are much more expansive thant docker :)

Reply:
but the way it is now, we would return an error if "docker:" was used. Should I change that?